src/type/__tests__/enumType.js

```diff
@@ -30,6 +30,13 @@ describe('Type System: Enum Values', () => {
     }
   });
 
+  var FavoriteType = new GraphQLObjectType({
```

Comment:
For unit tests, I prefer as simple a case as possible to illustrate the condition to be tested. Complexity can often obscure what is actually being tested.